vmm/sandbox/src/sandbox.rs

```diff
@@ -466,6 +466,10 @@ where
 
     async fn stop(&mut self, force: bool) -> Result<()> {
         match self.status {
+            // If sandbox is created but not running, no need to stop.
+            SandboxStatus::Created => {
```

Comment:
create <-> delete,   start <-> stop,  if we can make this clear, then maybe we don't need to call stop for Created sandbox, otherwise, maybe we have to stop the sandbox no matter whether it is running.

Reply:
containerd will call stop event start failed